cmd/zapi/cmd/post/post.go

```diff
@@ -116,6 +116,19 @@ func abspaths(paths []string) ([]string, error) {
 	var err error
 	out := make([]string, len(paths))
 	for i, path := range paths {
+		// Special handling for paths starting with S3:
+		// we will not convert to absolute path -MTW
+		if path[0:5] == "s3://" {
+			out[i] = path
+			continue
+		}
+		// TODO: This looks wrong to me,
```

Comment:
This is the current behavior because it was based around the Brim/zqd setup. The work in #1336 is a move towards not passing file paths at all to zqd.